components/HeroContent.js

```diff
@@ -0,0 +1,69 @@
+import css from "styled-jsx/css";
+
+export default props => (
+  <div className="hero-wrapper">
+    <div className="hero">
+      <div className="hero-content">
+        <h1>{props.title}</h1>
+        <p>{props.subtitle}</p>
+      </div>
+      <picture className="hero-media">
+        <img
+          className="hero-image"
+          role="presentation"
+          alt=""
+          src={props.mkUrl ? props.mkUrl(props.imageUrl) : props.imageUrl}
+        />
+      </picture>
+    </div>
+    <style jsx>{styles}</style>
+  </div>
+);
+
+let styles = css`
```

Comment:
Maybe styles should be put in a separate file? to keep things consistent

Reply:
uff da! I would not agree that we have "module-level indirection" for reasons of consistency. but sure.